core/connection.py

```diff
@@ -1,5 +1,9 @@
 from mongoengine import connect
 
+from os import getenv
 from core.config import HOST, PORT, DB_NAME
 
+if getenv('ENVIRONMENT') == 'docker':
```

Comment:
`if getenv('ENVIRONMENT') != 'docker':` ? otherwise mongo fails to be found

Reply:
Fixed